base/complex.jl

```diff
@@ -360,8 +360,8 @@ inv(z::Complex{<:Union{Float16,Float32}}) =
 #             c + i*d
 function /(z::ComplexF64, w::ComplexF64)
     a, b = reim(z); c, d = reim(w)
-    ab = max(abs(a), abs(b))
-    cd = max(abs(c), abs(d))
+    @fastmath ab = max(abs(a), abs(b))
```

Comment:
Note `@fastmath` isn't really valid here: it implies that we can assume `a` or `b` are never `Inf`, `NaN` or subnormal.

Reply:
Ah, you're quick. I thought I could already put my lessons above to use.

Reply:
Maybe I'm misunderstanding the nature of the @fastmath macro here: I thought it simply swapped the `max` function for [`max_fast`](https://github.com/JuliaLang/julia/blob/3fbdc9bbc97e4e5b1d682883ad9d14be5d65c862/base/fastmath.jl#L309) (and same for `abs`, though `abs` and `abs_fast` appear identical)?

Reply:
Regardless, I'll swap it to your initial suggestion instead, thanks.

Reply:
`@fastmath` is probably okay in this case, but occasionally it can cause some problems when the compiler gets carried away. Probably better to be explicit.